### ISSUE TITLE
Failed with "make info"

### ISSUE THREAD

**Issue by YorkZ** [kovidgoyal/kitty#5585]
**Describe the bug**
Failed when running `make info`

**To Reproduce**
1. `cd` into the latest `kitty` source tree cloned from GitHub
2. Run `make`
3. `cd doc`
4. `make info`

```
# Sphinx version: 5.1.1
# Python version: 3.10.7 (CPython)
# Docutils version: 0.18.1 release
# Jinja2 version: 3.1.2
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "~/.virtualenvs/python-3.10.5/lib/python3.10/site-packages/sphinx/cmd/build.py", line 272, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "~/.virtualenvs/python-3.10.5/lib/python3.10/site-packages/sphinx/application.py", line 236, in __init__
    self.config.setup(self)
  File "~/src/kitty/docs/conf.py", line 541, in setup
    write_cli_docs(kn)
  File "~/src/kitty/docs/conf.py", line 259, in write_cli_docs
    data = get_kitten_cli_docs(kitten)
  File "~/src/kitty/kittens/runner.py", line 151, in get_kitten_cli_docs
    run_kitten(kitten, run_name='__doc__')
  File "~/src/kitty/kittens/runner.py", line 112, in run_kitten
    runpy.run_module(f'kittens.{kitten}.main', run_name=run_name)
  File "/usr/lib/python3.10/runpy.py", line 220, in run_module
    mod_name, mod_spec, code = _get_module_details(mod_name)
  File "/usr/lib/python3.10/runpy.py", line 140, in _get_module_details
    raise error("No module named %s" % mod_name)
ImportError: No module named kittens.key_demo.main
```

**Environment details**
```
Press Ctrl+Shift+F6 (cmd+option+comma on macOS) in kitty, to copy debug output about kitty and its
configuration to the clipboard and paste it here. 

On older versions of kitty, run kitty --debug-config instead

kitty 0.26.3 created by Kovid Goyal
Linux york-asus 5.19.13-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 04 Oct 2022 14:36:58 +0000 x86_64
IP: 4{eno2}
Arch Linux 5.19.13-arch1-1 (/dev/tty)

DISTRIB_ID="Arch"
DISTRIB_RELEASE="rolling"
DISTRIB_DESCRIPTION="Arch Linux"
Running under: X11
Frozen: False
Paths:
  kitty: /usr/bin/kitty
  base dir: /usr/lib/kitty
  extensions dir: /usr/lib/kitty/kitty
  system shell: /bin/bash
Loaded config files:
  ~/.config/kitty/kitty.conf

Config options different from defaults:
allow_remote_control yes
font_size            10.0
listen_on            unix:@mykitty
scrollback_lines     262144
shell                /bin/fish
startup_session      ~/.config/kitty/home
Added shortcuts:
	alt+h →  launch --type=os-window --stdin-source=@screen_scrollback vim -R + -
	alt+j →  scroll_line_down
	alt+k →  scroll_line_up
	alt+n →  next_window
	alt+p →  previous_window
	alt+t →  next_layout
	ctrl+, →  move_tab_backward
	ctrl+- →  change_font_size current -2.0
	ctrl+. →  move_tab_forward
	ctrl+= →  change_font_size current +2.0
	ctrl+alt+n →  last_used_layout
	ctrl+alt+s →  goto_layout stack
	ctrl+alt+t →  goto_layout tall
	ctrl+alt+w →  set_tab_title
	ctrl+backspace →  change_font_size current 0
	ctrl+down →  scroll_line_down
	ctrl+end →  scroll_end
	ctrl+home →  scroll_home
	ctrl+left →  previous_tab
	ctrl+page_down →  scroll_page_down
	ctrl+page_up →  scroll_page_up
	ctrl+q →  close_tab
	ctrl+right →  next_tab
	ctrl+up →  scroll_line_up
	ctrl+v →  paste_from_clipboard

Important environment variables seen by the kitty process:
	PATH                                /usr/lib/ccache/bin:~/.local/bin:~/bin:~/.local/share/gem/ruby/2.7.0/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/sbin:/sbin:/bin:~/.cask/bin
	LANG                                en_US.UTF-8
	EDITOR                              emacsclient -c
	SHELL                               /bin/bash
	DISPLAY                             :0.0
	USER                                yzhao
	XDG_SEAT                            seat0
	XDG_SESSION_TYPE                    tty
	XDG_SESSION_CLASS                   user
	XDG_VTNR                            1
	XDG_SESSION_ID                      1
	XDG_RUNTIME_DIR                     /run/user/1000

```

**Additional context**
Try to reproduce the problem with `kitty --config NONE` if you cannot then post a minimal kitty.conf that reproduces the problem. If the problem involves interaction with some other terminal program post a minimal config for that program to reproduce the problem as well.

**Comment by page-down** [comment]
I can' t reproduce the issue under the master branch.

```shell
make
make -C docs info
```

It looks like you created your own new module under the `kittens` folder, so fix your code.
Also, there is no `doc` folder under the master branch, so `cd doc` will not work.

**Comment by YorkZ** [comment]
> It looks like you created your own new module under the kittens folder, so fix your code.

I don't think I had ever created any new module under the kittens folder. But I'll try it on a fresh new clone anyway.

> Also, there is no doc folder under the master branch, so cd doc will not work.

Sorry for my typo, and yes it should be `docs` not `doc`, and what I actually did was `cd docs`.

**Comment by YorkZ** [comment]
@page-down Running:

``` shell
rm -rf kitty
git clone git@github.com:kovidgoyal/kitty.git
cd kitty
make
make -C docs info
```

succeeded. Thanks for the help.

However, I have no idea why it failed. I cloned this repo about two years ago, and `make -C docs info` succeeded at that time. Yesterday I just wanted to update my info files, and therefore what I did was:

``` shell
cd kitty
git pull
make clean
make
cd docs
make info
```

I also checked `git status` which didn't show any new untracked files. Anyways, thanks again for the help.
